pkg/security/secl/eval/eval_test.go

```diff
@@ -269,7 +269,6 @@ func TestRegexp(t *testing.T) {
 		{Expr: `process.name !~ "/usr/sbin/*"`, Expected: true},
 		{Expr: `process.name =~ "/bin/"`, Expected: false},
 		{Expr: `process.name =~ "/bin/*"`, Expected: false},
```

Comment:
Might not be relevant but we can probably add new patterns now:
`{Expr: `process.name =~ "*/bin/*"`, Expected: true}`
`{Expr: `process.name =~ "*/bin"`, Expected: false}`
`{Expr: `process.name =~ "/usr/*/c$t"`, Expected: true}`
`{Expr: `process.name =~ "/usr/*/bin/*"`, Expected: true}`

Reply:
done